bears/general/TextLintBear.py

```diff
@@ -0,0 +1,242 @@
+import json
+
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.settings.Setting import typed_list
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+
+
+@linter(executable='textlint',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):(?P<column>\d+)(?:\s|\u2713)*'
```

Comment:
don't forget that we use raw-strings :D `\u2713` will be rendered here as `\u2713` actually and not `✓` :)